src/usernotifications.cs

```diff
@@ -382,10 +437,12 @@ interface UNNotificationContent : NSCopying, NSMutableCopying, NSSecureCoding {
 		NSDictionary UserInfo { get; }
 
 		[NoWatch, NoTV, iOS (12, 0)]
+		[Deprecated (PlatformName.iOS, 15, 0, message: "The property 'summaryArgument' is ignored.")]
```

Comment:
Wrong casing: our binding is `SummaryArgument`

Reply:
There's not much value in naming the property... and it makes the assembly larger
Simply use `"This property is ignored."`